src/pages/jonghoLee/LoginPage/LoginForm.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react'
+import { withRouter } from 'react-router-dom';
+import '../reset.css';
+import '../common.scss';
+import './LoginForm.scss';
+
+class LoginForm extends Component {
+  constructor() {
+    super();
+
+    this.state = {
+      login_id: "",
+      login_pw: "",
+      isAbled: true,
+    };
+  }
+
+  goToMain = (e) => {
+    e.preventDefault()
+    this.props.history.push('/main');
+  }
+
+  handleInput = (e) => {
+    this.setState({
+      [e.target.id]: e.target.value,
+    }, () => {
+      this.loginCheck();
+    });
+  }
+
+  loginCheck = () => {
+    const { login_id, login_pw } = this.state;
+    this.setState({ isAbled: !(login_id.includes("@") && login_pw.length >= 5) })
+  };
```

Comment:
```suggestion
  loginCheck = () => {
    const { login_id, login_pw } = this.state;
    const isAbled = login_id.includes("@") && login_pw.length >= 5;
    
    this.setState({ isAbled })
  };
```
- 인자가 길어지는 경우 변수를 활용해서 조금 더 깔끔하게 처리해볼 수 있습니다.
- 객체 단축 속성명을 사용하여 key, value 값이 동일한 경우 위와 같이 처리할 수 있습니다.

Reply:
넵 그렇게했브니다.